WindowsPackages/CreatingWindowsPackage.md

```diff
@@ -0,0 +1,20 @@
+## Creating Windows10/Server2016 MSI Package
```

Comment:
I agree. These instructions should be put in the `Windows` section in `docs/maintainers/releasing.md`.